src/test/resources/capabilities-bstack-single.yml

```diff
@@ -0,0 +1,40 @@
+testEndpoint: https://bstackdemo.com/
+
+driverType: remoteDriver
+
+localDriver:
```

Comment:
LocalDriver shouldn't be there for this profile?

Reply:
Removed Local Driver